src/main/kotlin/no/nav/syfo/narmestelederrelasjon/NarmesteLederRelasjonService.kt

```diff
@@ -1,20 +1,39 @@
 package no.nav.syfo.narmestelederrelasjon
 
 import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.client.pdl.PdlClient
 import no.nav.syfo.domain.PersonIdentNumber
 import no.nav.syfo.narmestelederrelasjon.database.domain.toNarmesteLederRelasjonList
 import no.nav.syfo.narmestelederrelasjon.database.getNarmesteLederRelasjonList
 import no.nav.syfo.narmestelederrelasjon.domain.NarmesteLederRelasjon
+import no.nav.syfo.narmestelederrelasjon.domain.addNarmesteLederName
 
 class NarmesteLederRelasjonService(
     private val database: DatabaseInterface,
+    private val pdlClient: PdlClient,
 ) {
-    fun getRelasjonerForPersonIdent(
-        personIdentNumber: PersonIdentNumber
+    suspend fun getRelasjonerForPersonIdent(
+        callId: String,
+        personIdentNumber: PersonIdentNumber,
     ): List<NarmesteLederRelasjon> {
-        val pNarmesteLederRelasjonList = database.getNarmesteLederRelasjonList(
+        val narmesteLederRelasjonList = database.getNarmesteLederRelasjonList(
             personIdentNumber = personIdentNumber,
-        )
-        return pNarmesteLederRelasjonList.toNarmesteLederRelasjonList()
+        ).toNarmesteLederRelasjonList()
+
+        if (narmesteLederRelasjonList.isEmpty()) {
```

Comment:
Hadde det vært bedre om pdlClient bare returnerte tomt Map (eller null) hvis den fikk en tom liste ? i stedet for at de som bruker pdlClient må sjekke tom liste for å unngå kall til PDL

Reply:
Høres greit ut det, har endret til dette nå:
```
        if (personIdentNumberList.isEmpty()) {
            return emptyMap()
        }
        ```